LICENSE.BSD

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
```

Comment:
Could we please rename that file LICENSE-BSD-3-Clause and the Apache one LICENSE-APACHE? Just to be consistent with the already existing crates?

Reply:
Will do that.